doc/source/user_guide/new_meshing_workflows.rst

```diff
@@ -0,0 +1,462 @@
+.. _ref_user_guide_new_meshing_workflows:
+
+Use meshing workflows
+=====================
+PyFluent supports accessing all Fluent meshing functionalities, including
+guided meshing workflows.
+
+Watertight geometry meshing workflow
+------------------------------------
+This simple example shows how to use the watertight geometry meshing workflow.
+
+Import geometry
+~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    import ansys.fluent.core as pyfluent
+    from ansys.fluent.core import examples
+
+    import_file_name = examples.download_file('mixing_elbow.pmdb', 'pyfluent/mixing_elbow')
+    meshing = pyfluent.launch_fluent(
+        mode="meshing", precision='double', processor_count=2
+    )
+    watertight = meshing.watertight()
+    watertight.import_geometry.FileName.set_state(import_file_name)
+    watertight.import_geometry.LengthUnit.set_state('in')
+    watertight.import_geometry.Execute()
```

Comment:
```suggestion
    watertight.import_geometry()
```
This should work and it is the intended usage. We could eliminate `Execute` in this mode.